src/rent/extrinsics.ts

```diff
@@ -147,56 +152,62 @@ export const revokeContract = async (
 
 /**
  * @name rentTx
- * @summary                Creates an unsigned unsubmitted Rent-Contract Transaction Hash for an NFT.
- * @param  nftId           The NFT Id with the contract to rent.
- * @returns                Unsigned unsubmitted Rent-Contract Transaction Hash. The Hash is only valid for 5 minutes.
+ * @summary                       Creates an unsigned unsubmitted Rent-Contract Transaction Hash for an NFT.
+ * @param nftId                   The NFT Id with the contract to rent.
+ * @param signedRentStartBlockId  The contract creation block id to check to ensure contract authenticity.
```

Comment:
* @param signedRentStartBlockId The contract creation block id to check to ensure contract authenticity.
Here the name signedRentStartBlockId should be contractCreationBlockId because contract is not started yet.